### ISSUE TITLE
Improve contrast ratio

### ISSUE THREAD

**Issue by denar90** [marionettejs/marionettejs.com#495]
We have not so good contrast ratio. Using e.g. [Lighthouse](https://github.com/GoogleChrome/lighthouse/), we have these results

![image](https://cloud.githubusercontent.com/assets/6231516/24085080/eab46800-0cfd-11e7-9465-7788d6bf2f7f.png)

P.S. 

https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd
http://leaverou.github.io/contrast-ratio/ 

might help to improve it

**Comment by peterblazejewicz** [comment]
This extension (from Google) adds an option to *audits* section in dev tools to verify a11y rules. It procies hinted color that should be used instead of one failing tests based on luminosity change. So you could do adjustment pretty mach based on this tool hints:
https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb?hl=en
The footer is secondary content (always hidden under the fold), so this should be OK to just fix color using hints (for links)
The masthead buttons background color is the other thing. Just check what you get with a11y tool

**Comment by denar90** [comment]
thx 👍

**Comment by paulfalgout** [comment]
I think we should just ask @samccone to tweak lighthouse so marionettejs.com passes.  :1st_place_medal:

**Comment by samccone** [comment]
:)  thanks @peterblazejewicz for the detailed response. in this case I think no magic lighthouse hacks are needed :P
